### DISCUSSION TITLE
[15주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#106] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 이동호
* [Network] REST - API #103
2. 김정윤
* [ETC] 폭포수 방법론과 애자일 방법론 #101
3. 이주형
* [OS] 메모리 관리 #105

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 🍀
* 좋았던 점
* 건의사항
* 느낀점

**Reply by leeejuhyeong** [reply]
다음은 비연속 메모리 할당...
고생하셨습니다~

**Reply by KJY97** [reply]
OS는 역시 어렵네요.. 수고하셨습니다

**Reply by doho-ho** [reply]
다음은 REST API 어떻게 쓰는지~~~ os 넘 어렵네요,,.,
